src/floor.ts

```diff
@@ -0,0 +1,36 @@
+import { _round } from './_round';
+import { purry } from './purry';
+
+/**
+ * Rounds down a given number to a specific precision.
+ * @param value The number to round down.
+ * @param precision The precision to round down to. Must be an integer between -15 and 15.
+ * @signature
+ *    R.floor(value, precision);
+ * @example
+ *    R.floor(123.4317, 3) // => 123.431
+ *    R.floor(483.22243, 1) // => 483.2
+ *    R.floor(8541, -2) // => 8500
```

Comment:
All these examples need to show the difference between floor, ceil, and round. I would take an example where each one of them would have a different result. Assume some users don't know what `ciel` and `floor` do.

e.g.
```suggestion
 *    R.floor(123.9876, 3) // => 123.987
 *    R.floor(483.22243, 1) // => 483.2
 *    R.floor(8541, -1) // => 8540
 *    R.floor(456789, -3) // => 456000
```

For `round`, these examples would look like this:
```ts
 *    R.round(123.9876, 3) // => 123.988
 *    R.round(483.22243, 1) // => 483.2
 *    R.round(8541, -1) // => 8540
 *    R.round(456789, -3) // => 457000
```

And for `ceil`:
```ts
 *    R.ceil(123.9876, 3) // => 123.987
 *    R.ceil(483.22243, 1) // => 483.3
 *    R.ceil(8541, -1) // => 8550
 *    R.ceil(456789, -3) // => 456000
```

This way users can compare the output and pick the right one for their use-case.

Reply:
I agree and that was my intention when providing these examples, although I can see how your examples give a clearer view on the differences between these functions than the ones I provided (particularly the negative precision ones).